collections/_release_4_4/entry-highlights-ubershaders.md

```diff
@@ -0,0 +1,29 @@
+---
+type: entry
+section: highlights
+rank: 4
+importance: 2
+anchor: ubershaders
+title: Ubershaders to reduce stutter
+blockquote: <span class="highlight">uber</span>-exciting!
+text: |
+  Ubershaders are at load time pre-compiled versions of each shader with all their features.
+
+  These shaders becomes the fallback whenever a new object is drawn, so that freezing - as we wait for the more specialized shader pipeline in the background to compile - becomes a thing of the past. This technique therefore completely avoids shader stutter!
```

Comment:
```suggestion
  These shaders become the fallback whenever a new object is drawn, so that freezing - as we wait for the more specialized shader pipeline in the background to compile - becomes a thing of the past. This technique therefore completely avoids shader stutter!
```